src/FixedDataTable.js

```diff
@@ -613,6 +642,10 @@ class FixedDataTable extends React.Component {
     this._contentHeight = contentHeight;
   }
 
+  shouldComponentUpdate(nextProps) {
```

Comment:
not specific to this PR, in general good to have.

Reply:
FixedDataTable has a few objects being passed as props. So I suggest removing the `shouldComponentUpdate` check here, especially since it's not required by the PR.

If we are absolutely certain that the passed in objects are immutable and that they won't change in the future, then we could have this in.